DataFormats/METReco/interface/CSCHaloData.h

```diff
@@ -69,6 +78,20 @@ namespace reco {
     edm::RefVector<reco::TrackCollection>& GetTracks(){return TheTrackRefs;}
     const edm::RefVector<reco::TrackCollection>& GetTracks()const {return TheTrackRefs;}
     
+    edm::RefVector<EcalRecHitCollection>& GetEBRechits(){return ecalebrhRefs;}
```

Comment:
It doesn't seem appropriate to extend CSCHaloData with calorimeter information.
The design of the halo data formats was to collect detector-specific information first and then have a combination in GlobalHaloData (or in BeamHaloSummary)